live-examples/js-examples/intl-numberformat-supportedlocalesof.html

```diff
@@ -0,0 +1,8 @@
+<pre>
+<code id="static-js">var locales1 = ['ban', 'id-u-co-pinyin', 'de-ID'];
+var options1 = { localeMatcher: 'lookup' };
+
+console.log(Intl.NumberFormat.supportedLocalesOf(locales1, options1));
+// expected output: Array ["id-u-co-pinyin", "de-ID"]
+</code>
```

Comment:
As for `Intl.Collator.supportedLocalesOf()`, I guess this is browser-dependent?

Reply:
Yup.